main.go

```diff
@@ -91,6 +91,8 @@ func main() {
 	flagSet.String("redis-connection-url", "", "URL of redis server for redis session storage (eg: redis://HOST[:PORT])")
 	flagSet.Bool("redis-use-sentinel", false, "Connect to redis via sentinels. Must set --redis-sentinel-master-name and --redis-sentinel-connection-urls to use this feature")
 	flagSet.String("redis-sentinel-master-name", "", "Redis sentinel master name. Used in conjunction with --redis-use-sentinel")
+	flagSet.String("redis-ca-path", "", "Redis custom CA path")
+	flagSet.Bool("redis-insecure-tls", false, "Use insecure TLS connection to redis")
```

Comment:
Potential improvement for making this more verbose? (insecure stuff should be verbose IMO so it's obvious to people)
```suggestion
	flagSet.Bool("redis-insecure-skip-tls-verify", false, "Use insecure TLS connection to redis")
```
It would also then match the code a bit more, and I've seen flags called `insecure-skip-tls-verify` many times so it may be more idiomatic? 🤔